tests/test_np.py

```diff
@@ -367,17 +370,105 @@ def test_empty():
 		assert_equal(loads(json), data, 'shape = {} ; json = {}'.format(data.shape, json))
 
 def test_decode_writeable():
-    # issue https://github.com/mverleg/pyjson_tricks/issues/90
-    data = zeros((2, 2))
+	# issue https://github.com/mverleg/pyjson_tricks/issues/90
```

Comment:
I can undo the apparent changes to this test, however, I feel like the whole tab / spaces war must be resolved. 

I've tried to follow "use tabs" as much as a I could with my editors.

Happy to work to revert these changes once important changes  are approved.

Reply:
It's fine this way I think